src/resultsdisassemblypage.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+  resultsdisassemblypage.cpp
+
+  This file is part of Hotspot, the Qt GUI for performance analysis.
+
+  Copyright (C) 2017-2020 Klarälvdalens Datakonsult AB, a KDAB Group company, info@kdab.com
+  Author: Nate Rogers <nate.rogers@kdab.com>
+
+  Licensees holding valid commercial KDAB Hotspot licenses may use this file in
+  accordance with Hotspot Commercial License Agreement provided with the Software.
+
+  Contact info@kdab.com if any conditions of this licensing are not clear to you.
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 2 of the License, or
+  (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "resultsdisassemblypage.h"
+#include "ui_resultsdisassemblypage.h"
+
+#include <QMenu>
+#include <QFileDialog>
+#include <QFile>
+#include <QTextStream>
+#include <QMessageBox>
+#include <QString>
+#include <QListWidgetItem>
+#include <QProcess>
+#include <QDebug>
+#include <QDir>
+#include <QDirIterator>
+
+#include <KRecursiveFilterProxyModel>
+
+#include "parsers/perf/perfparser.h"
+#include "resultsutil.h"
+
+#include "models/costdelegate.h"
+#include "models/hashmodel.h"
+#include "models/topproxy.h"
+#include "models/treemodel.h"
+
+#include <QStandardItemModel>
+
+ResultsDisassemblyPage::ResultsDisassemblyPage(FilterAndZoomStack *filterStack, PerfParser *parser, QWidget *parent)
+        : QWidget(parent), ui(new Ui::ResultsDisassemblyPage) {
```

Comment:
is this what you got from clang-format? I don't think so, or? it should be:

````
Foo::Foo(...)
    : a(...)
    , b(...)
{
    ...
}
```

Reply:
Hm, I have run git-clang-format HEAD^ before commit and then commit as you asked. Ok, I will fix this manually.

Reply:
if this is clang-format clean, then leave it as is, I'll have to fix the clang format style file then